clap-v3-utils/Cargo.toml

```diff
@@ -15,6 +15,10 @@ targets = ["x86_64-unknown-linux-gnu"]
 [lib]
 name = "solana_clap_v3_utils"
 
+[features]
+default = ["elgamal"]
```

Comment:
Since we're in the "breaking" phase in v3.0.0, we can potentially make this not a default feature.

Since there aren't too many users, it might be better to _not_ make it default, but I don't feel too strongly about it.

Reply:
Yeah I was considering this as well, but was afraid of the breaking change. I removed `elgamal` from defaut.